session-1/coba.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- CSS Bootsrtap -->
+    <link rel="stylesheet" href="assets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="assets/css/main.css">
+    <title>Experience</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg fixed-top navbar-light bg-light border-bottom shadow-lg">
+            <div class="container">
+                <a class="navbar-brand" href="#">
+                    <h1 class="display-6"><img src="assets/img/Wikimedia-logo.png" alt="" width="50" class="d-inline-block align-text-top"> Danjou </h1>
+                </a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+                <div class="collapse navbar-collapse" id="navbarNav">
+                    <ul class="navbar-nav ms-auto">
+                        <li class="nav-item me-2">
+                            <a class="nav-link active" aria-current="page" href="#">About</a>
+                        </li>
+                        <li class="nav-item me-2">
+                            <a class="nav-link active" href="#">Experience</a>
+                        </li>
+                        <li class="nav-item me-3">
+                            <a class="nav-link active" href="#">Skills</a>
+                        </li>
+                        <li class="nav-item me-3">
+                            <a class="nav-link active" href="#">Portofolio</a>
+                        </li>
+                        <li class="nav-item me-3">
+                            <a href="#" class="nav-link active btn btn-success rounded" style="color: #fff;">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <!-- Content -->
+    <main class="p-5 py-5 mt-5">
+        <h1 class="text-center text-decoration-underline mt-3 fw-bold">Experience</h1>
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6 col-sm-6 py-3 align-self-center">
```

Comment:
Add col-12 to make mobile version, combine with col-lg col-xl col-md. Read this documentation [Column](https://getbootstrap.com/docs/5.1/layout/columns/)